src/js/function.js

```diff
@@ -0,0 +1,8 @@
+function Exams(data, marks) {
+  console.log(`Results:
+  Subject: ${data.js}, mark: ${marks[0]},
+  Subject: ${data.html}, mark: ${marks[1]},
+  Subject: ${data.css}, mark: ${marks[2]}.`);
+}
+
+export { Exams };
```

Comment:
please use named export here. You can just export `Exams ` without curly braces

Reply:
Done